frontend/src/pages/mock/logout.page.tsx

```diff
@@ -23,7 +25,11 @@ const MockLogout: NextPage = () => {
       method: "GET",
       redirect: "manual",
     });
-    // Revalidate account data to reflect logged out status:
+    // Revalidate account data to reflect logged out status.
+    // Note that the `clear` method exists, even though it's not exposed on the
+    // type yet:
+    // https://github.com/vercel/swr/issues/161#issuecomment-1079198998
```

Comment:
praise: thanks for sending a PR upstream! I subscribed to it to hopefully help remember when we can remove this work-around.